packages/rules-unit-testing/api-extractor.json

```diff
@@ -1,5 +1,5 @@
 {
     "extends": "../../config/api-extractor.json",
-    // Point it to your entry point d.ts file. 
-    "mainEntryPointFilePath": "<projectFolder>/dist/rules-unit-testing/index.d.ts"
-}
\ No newline at end of file
+    // Point it to your entry point d.ts file.
```

Comment:
BTW, I needed to make this change otherwise the tool wouldn't run correctly. Should I leave it in the PR or remove it?

Reply:
Sorry about this again, this might be because of an updated rollup plugin dependency that was merged after you made this branch. The old plugin would write to the path you've updated it to, the new version writes to the new path - this change is causing the docgen step to fail in CI now.

Locally can you try deleting node_modules and reinstalling deps?

```
// in the root of the repo
rm -fr node_modules
yarn
// try docgen again
yarn docgen:all
```